dev/tests/integration/testsuite/Magento/MessageQueue/Model/ResourceModel/LockTest.php

```diff
@@ -0,0 +1,28 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\MessageQueue\Model\ResourceModel;
+
+use Magento\Framework\MessageQueue\LockInterface;
+use Magento\TestFramework\ObjectManager;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * Covers Lock resource model test cases
+ */
+class LockTest extends TestCase
+{
+    public function testSaveLock()
```

Comment:
This test doesn't cover initial test case. Please add more specific test.

Reply:
added test that covers original case